Jenkinsfile

```diff
@@ -35,25 +35,39 @@ pipeline {
                         env.awsTokenId = 'aws-sandbox-token'
                         env.awsProfile = 'uoa-sandbox'
                         env.awsAccountId = '416527880812'
+                        withCredentials([
```

Comment:
At the moment we're only declaring `env` variables here that are used *regardless* of the project that is being built (e.g. we need `awsTokenId/awsProfile` for all 3 projects, while the variables we'll only need conditionally we're only getting in the steps we need them, e.g.:

https://github.com/UoA-eResearch/hub-stack/blob/5a6a4fe6d858a7b6b47b82230064151b4ebb57a2/Jenkinsfile#L180-L189

Do you think maybe we should put this stuff in the `build` step of the `cer-graphql` project?

Reply:
Good idea :+1:, will put it in build and test steps.

Reply:
I have now moved this into the `build` stage. I also tried putting it in the `test` stage, since it also requires the credentials file, but Jenkins gave me a permission denied error. I removed it since `test` stage precedes `build`, so the file will already be copied into it. 
At the moment the build stage will always be run if the test stage is run, so we can always rely on the credentials file being copied at `build` stage. If this changes we will need to revisit this.